### PR TITLE
SDA-3237 Notifications re-design

### PR DESCRIPTION
**Pull request by sbenmoussati**
## Description
Global re-design of notifications:
- Layout
- Adding flashing capabilities which changes (external, mention etc.)

[JIRA Ticket](https://perzoinc.atlassian.net/browse/SDA-3237)

[Figma File](https://www.figma.com/file/LxEcEuY7BME56Iunk4AXgo/SDA-notifications-improvements?node-id=86%3A1902)

### CODE REVIEW

File: spec/notificationComp.spec.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    externalBadge = wrapper.find('.ext-badge-container');
+    expect(externalBadge.exists()).toBeTruthy();
+  });
+
+  it('should flash as a mention when mention sent', () => {
+    const theme = Themes.DARK;
+    const flash = true;
+    const hasMention = true;
+    const themedMentionFlashing = `.${theme}-mention-flashing`;
+    let themedToastNotification = wrapper.find(themedMentionFlashing);
```
Comment (psjostrom) [inline spec/notificationComp.spec.ts:78]: Purely personal coding style preference, but why not have everything as const, and leave mutation completely out of the picture?

File: src/renderer/components/notification-comp.tsx [hunk 2]
```diff
@@ -318,15 +361,6 @@ export default class NotificationComp extends React.Component<{}, IState> {
     }
   }
 
-  /**
-   * Clears a active notification flash interval
-   */
-  private clearFlashInterval(): void {
```
Comment (psjostrom) [inline src/renderer/components/notification-comp.tsx:324]: So we had support for flashing notifs already?

**Review by psjostrom — APPROVED** [review]
Impressive, LGTM

File: spec/notificationComp.spec.ts (hunk 1 above)
Comment (sbenmoussati) [reply spec/notificationComp.spec.ts:78]: dummy copy/paste, will change that.

File: src/renderer/components/notification-comp.tsx (hunk 2 above)
Comment (sbenmoussati) [reply src/renderer/components/notification-comp.tsx:324]: well spotted.. I had exactly the same comment when I started digging into it but I didn't find anything related to flashing in C2 code base and I don't think we have custom notifications in 1.5

**Review by cedric-lal — APPROVED** [review]

**Comment by cedric-lal** [comment]
Seems like I don't have write access to SymphonyElectron

**Review by tarpintin-symphony — APPROVED** [review]

**Pull request merged**